big_tests/tests/graphql_gdpr_SUITE.erl

```diff
@@ -49,38 +55,25 @@ admin_gdpr_test(Config) ->
 
 admin_gdpr_test(Config, Alice) ->
```

Comment:
Please change the test suit name because it will be very similar to the group name. Maybe `admin_retrive_user_data`.